lib/ttnt/testtask.rb

```diff
@@ -92,8 +110,7 @@ def define_anchor_task
             "-I#{gem_root} -r ttnt/anchor " +
             "#{@rake_testtask.ruby_opts_string}"
 
-          test_files = Rake::FileList[@rake_testtask.pattern].compact
-          test_files += @test_files.to_a if @test_files
+          test_files = expanded_file_list
```

Comment:
Oops, thank you for catching. Fixed here: https://github.com/Genki-S/ttnt/commit/e4a741a92bae5f08a60ad7cf9ca9e9d89d0b3f06